src/App.js

```diff
@@ -1,32 +1,25 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import { Provider } from 'react-redux';
 import Counter from '../src/components/Counter';
-import {
-  BrowserRouter as Router,
-  Switch,
-  Route,
-} from 'react-router-dom';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import store from './redux/store';
 
-const Home = () => <h1> Dummy Home Page</h1>;
-const Login = () => <h1> Dummy Login Page </h1>;
+import SignIn from './views/SignIn';
 
 class App extends Component {
-	render(){
+	render() {
 		return (
-	 <Provider store = {store}>
-        <Router>
-      <div className="center">
-             <Switch>
-             <Route path='/' exact component={Home} />
-             <Route path='/signin' exact component={Login}/>
-             <Route path='/count' exact component={Counter}/>
-             </Switch>
-      </div> 
-      </Router>
-     </Provider>
-	  );
-  }
+			<Provider store={store}>
+				<Router>
+					<div className="center">
+						<Switch>
+							<Route path="/" exact component={SignIn} />
+						</Switch>
+					</div>
+				</Router>
```

Comment:
Create a file where you will put all routes. Routes don't have to be in this file

Reply:
yeah let me do so